.circleci/config.yml

```
@@ -45,27 +45,27 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker
-      - run:
+      - run: # this is only to satisfy poetry; not used
           working_directory: ~/
           name: Clone pyobas
-          command: |
-            if [ "${CIRCLE_BRANCH}" = "release/current" ]; then
-              git clone -b release/current https://github.com/OpenBAS-Platform/client-python
-            else
-              git clone -b main https://github.com/OpenBAS-Platform/client-python
-            fi
+          command: git clone -b release/current https://github.com/OpenBAS-Platform/client-python
```

as it is only to satisfy poetry can we clone main instead ?
```suggestion
          command: git clone -b main https://github.com/OpenBAS-Platform/client-python
```